### ISSUE TITLE
Enable accounts with advanced features

### ISSUE THREAD

**Issue by sharknetwork24** [WWBN/AVideo#5225]
Could you tell me how to enable a user with all the advanced features?

I can't remember the command line steps anymore

![Cattura](https://user-images.githubusercontent.com/82677269/119734411-3e469500-be7b-11eb-9f08-5bb2cce6162a.PNG)

**Comment by DanielnetoDotCom** [comment]
Go to the encoder install folder and type this 

php makeAdmin.php

Then Type the username you want to make admin

if you cannot do it I will do for you later

**Comment by sharknetwork24** [comment]
cd /var/www/html/AVideo/Encoder/install
php makeAdmin.php

Enter the username and enter.

**Comment by DrDoug88** [comment]
I didn't have the Encoder install files. I did a git pull and attempted to get the Encoder directory in the right place. Anyhow, long story short, I completely forgot about the encoder database. I ended up just updating the admin user in the encoder database to have admin rights. The admin username was changed awhile back to enforce the unique email policy. So, I have updated it, and now I have proper admin access to the encoder site, and I can see all of the options correctly. Thanks for your help Daniel.
